docs/schemas/2024-10/donor-details.json

```diff
@@ -325,6 +333,23 @@
           "type": "string"
         }
       }
+    },
```

Comment:
Are you ditching "reference" altogether? Do you not have access to anything useful for One Login? If you're removing that field, we should probably also update the main validateConfirmIdentity method.

Reply:
I guess it could be removed totally until there is a need for it. I was assuming there was going to be something useful for the paper journey, but that is 🔮